trade-finance-manager-api/src/v1/controllers/amendment.controller.js

```diff
@@ -56,9 +56,35 @@ const sendAmendmentEmail = async (amendmentId, facilityId) => {
   }
 };
 
+// function to update tfm deals lastUpdated once amendment complete
+const updateTFMDealLastUpdated = async (amendmentId, facilityId) => {
+  const amendment = await api.getAmendmentById(facilityId, amendmentId);
+
+  if (amendment?.dealId) {
+    const { dealId } = amendment;
+    const payload = {
+      tfm: {
+        lastUpdated: new Date().valueOf(),
```

Comment:
does this return the time in milliseconds?

Reply:
yes.  It is stored originally in milliseconds on deal creation so have kept it the same.  Either way, I need to pass something as the update body and it gets overwritten by the same new Date().valueOf(); once in tfm-update-deal.controller.js